src/sourmash/tax/tax_utils.py

```diff
@@ -1679,50 +1208,64 @@ def load(cls, locations, **kwargs):
         return tax_assign
 
 
-# strategy from: https://subscription.packtpub.com/book/programming/9781800207455/10/ch10lvl1sec01/using-dataclasses-to-simplify-working-with-csv-files
 @dataclass
-class GatherRow(): # all cols should match "gather_write_cols" in `search.py`
-   # essential columns
-   query_name: str
-   name: str # match_name
-   f_unique_weighted: float
-   f_unique_to_query: float
-   unique_intersect_bp: int
-   remaining_bp: int
-   query_md5: str
-   query_filename: str
-   # new essential cols: requires 4.4x
-   query_bp: int
-   ksize: int
-   scaled: int
-
-   # non-essential
-   intersect_bp: int = None
-   f_orig_query: float = None
-   f_match: float = None
-   average_abund: float = None
-   median_abund: float = None
-   std_abund: float = None
-   filename: str = None
-   md5: str = None
-   f_match_orig: float = None
-   gather_result_rank: str = None
-   moltype: str = None
-   query_n_hashes: int = None
-   query_abundance: int = None
-   query_containment_ani: float = None
-   match_containment_ani: float = None
-   average_containment_ani: float = None
-   max_containment_ani: float = None
-   potential_false_negative: bool = None
-   n_unique_weighted_found: int = None
-   sum_weighted_found: int = None
-   total_weighted_hashes: int = None
+class GatherRow():
```

Comment:
```suggestion
class GatherRow:
```
note, `()` not needed!

Reply:
or maybe they are for dataclasses?

Reply:
nope, you're right - not needed. fixed!